deploy/12_deploy_coverage_pool_beneficiary.ts

```diff
@@ -6,7 +6,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { read } = deployments
```

Comment:
We'll have to confirm how the Treasury Guild wants to send rewards and if they want to do it via escrow. I suspect the answer is yes because they will probably mint some amount of tokens, fund the escrow, and then release rewards every week or so.

Reply:
Captured in #220